CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ option(CARE_NEVER_USE_RAJA_PARALLEL_SCAN "Disable RAJA parallel scans in SCAN lo
 #  practice for Release / RelWithDebInfo configurations.
 option(CARE_ENABLE_FUSER_BIN_32 "Enable the 32 register fusible loop bin." OFF)
 option(CARE_ENABLE_PARALLEL_LOOP_BACKWARDS "Reverse the start and end for parallel loops." OFF)
+option(CARE_ENABLE_STALE_DATA_CHECK "Enable checking for stale data." OFF)
```

Comment:
This is optional, but you could add a comment here saying that this option is only applicable for GPU or GPU simulation modes. 

On the other hand, that might be implied given that if you are only on the host you never transfer data so stale data would not be relevant. Up to you.

Reply:
Good idea! Done.